data/data-pipeline/data_pipeline/tests/score/test_utils/data/us.geojson

```diff
@@ -2,14 +2,14 @@
-76.811748, 39.296846 ], [ -76.812737, 39.296736 ], [ -76.813301, 39.29669 ], [ -76.814143, 39.296617 ], [ -76.815314, 39.296609 ], [ -76.816029, 39.29661 ], [ -76.816749, 39.29665 ], [ -76.817691, 39.296753 ], [ -76.818141, 39.296779 ], [ -76.818386, 39.296794 ], [ -76.818628, 39.296804 ], [ -76.818861, 39.296805 ], [ -76.81962, 39.296781 ], [ -76.820341, 39.29674 ], [ -76.821557, 39.296627 ], [ -76.822439, 39.296584 ], [ -76.823306, 39.296569 ], [ -76.824645, 39.296599 ], [ -76.824856, 39.296598 ], [ -76.825503, 39.296619 ], [ -76.82605, 39.296617 ], [ -76.826506, 39.296623 ], [ -76.833032, 39.296569 ], [ -76.834454, 39.296513 ], [ -76.835779, 39.296463 ], [ -76.837182, 39.296414 ], [ -76.840925, 39.296269 ], [ -76.846458, 39.296053 ], [ -76.847385, 39.296009 ], [ -76.848616, 39.295953 ], [ -76.85006, 39.295947 ], [ -76.851894, 39.295995 ], [ -76.853487, 39.296105 ], [ -76.854745, 39.296225 ], [ -76.855258, 39.29629 ], [ -76.855857, 39.29638 ], [ -76.856944, 39.296531 ], [ -76.857948, 39.296713 ], [ -76.85898, 39.296931 ], [ -76.859642, 39.297082 ], [ -76.8603, 39.297254 ], [ -76.861513, 39.297574 ], [ -76.862788, 39.29796 ], [ -76.864048, 39.298387 ], [ -76.865139, 39.298813 ], [ -76.86639, 39.299345 ], [ -76.867603, 39.299888 ], [ -76.86793, 39.300013 ], [ -76.868661, 39.300336 ], [ -76.869016, 39.300494 ], [ -76.869593, 39.300737 ], [ -76.870822, 39.301262 ], [ -76.87273, 39.302082 ], [ -76.87337, 39.302359 ], [ -76.874166, 39.302697 ], [ -76.874962, 39.303044 ], [ -76.875485, 39.303304 ], [ -76.875803, 39.303442 ], [ -76.876392, 39.303677 ], [ -76.876493, 39.303717 ], [ -76.876886, 39.303853 ], [ -76.877627, 39.304125 ], [ -76.878326, 39.304351 ], [ -76.878916, 39.304525 ], [ -76.879667, 39.304732 ], [ -76.880261, 39.304878 ], [ -76.880578, 39.304954 ], [ -76.88072, 39.304984 ], [ -76.880733, 39.305174 ], [ -76.880735, 39.305263 ], [ -76.880547, 39.305504 ], [ -76.879848, 39.305738 ], [ -76.87922, 39.305965 ], [ -76.877891, 39.306433 ], [ -76.877241, 39.306642 ], [ -76.87623, 39.306966 ], [ -76.875149, 39.307313 ], [ -76.874586, 39.3075 ], [ -76.870653, 39.308758 ], [ -76.87045, 39.308843 ], [ -76.870392, 39.30884 ], [ -76.869161, 39.308786 ], [ -76.869153, 39.308799 ], [ -76.869122, 39.308816 ], [ -76.869013, 39.308862 ], [ -76.868485, 39.309055 ], [ -76.867978, 39.309239 ], [ -76.867705, 39.309347 ], [ -76.867261, 39.309502 ], [ -76.866102, 39.309894 ], [ -76.866101, 39.309894 ], [ -76.865669, 39.310066 ], [ -76.864234, 39.310568 ], [ -76.863039, 39.310937 ], [ -76.862618, 39.311062 ], [ -76.86255, 39.311114 ], [ -76.862538, 39.311172 ], [ -76.862581, 39.311225 ], [ -76.862677, 39.311297 ], [ -76.862787, 39.311386 ], [ -76.862929, 39.311549 ], [ -76.863046, 39.311741 ], [ -76.863144, 39.311933 ], [ -76.863206, 39.312082 ], [ -76.863267, 39.312264 ], [ -76.86331, 39.31248 ], [ -76.86334, 39.312713 ], [ -76.863359, 39.313026 ], [ -76.863314, 39.313502 ], [ -76.863203, 39.314109 ], [ -76.863106, 39.314569 ], [ -76.863049, 39.31484 ] ] ] } },
```

Comment:
Note: all I changed here is renaming the field `GEOID10_TRACT` as `GEOID10`. This is more realistic. And also, it will otherwise yell at us now that I set `errors="raise"` in 

```
tract_data = tract_data.rename(
        columns={"GEOID10": "GEOID10_TRACT"}, errors="raise"
    )
 ```